docs/DeveloperGuide.md

```diff
@@ -50,7 +50,7 @@ There would be 4 settings for the priority feature: High, Medium, Low, Undefined
 
 ![Proposed UI](images/UIPriority.png)
 
-Firstly, the field would be added to the Person class. This would be similar to the method described in the tutorial:  https://nus-cs2103-ay2021s1.github.io/tp/tutorials/AddRemark.html, where instead of adding a command, we would be editing the add command to include an optional priority field. 
+Firstly, the field would be added to the Person class. This would be similar to the method described in the [tutorial](https://nus-cs2103-ay2021s1.github.io/tp/tutorials/AddRemark.html), where instead of adding a command, we would be editing the add command to include an optional priority field. 
```

Comment:
Also, can you format the classes/objects/variables too, for the parts that you are maintaining? E.g. `Person`

Reply:
kk

Reply:
Approving this first but formatting is pending